test/views/components/InputSpec.scala

```diff
@@ -121,7 +117,7 @@ class InputSpec extends WordSpec with Matchers with GuiceOneAppPerSuite {
       private val firstHint = hints.head
       private val hint1Attrs = elementAttrs(firstHint)
       hint1Attrs("class") shouldBe "govuk-hint"
-      firstHint.text() shouldBe Text(i1Hint).value(messages.lang).head.toString
+      firstHint.text() shouldBe Text(i1Hint).items.head.toString
```

Comment:
Is Text(i1Hint).items.head.toString a long winded way of saying i1Hint?

Reply:
done